bin/install-chrome-dependencies

```diff
@@ -65,19 +65,32 @@ case "${STACK}" in
     error "STACK must be 'heroku-20', 'heroku-22' or 'heroku-24', not '${STACK}'."
 esac
 
-if [ ! -f $CACHE_DIR/PURGED_CACHE_V1 ]; then
```

Comment:
I've removed this since it was a copy-paste from the old Chrome buildpack to handle a cache migration from 2018:
https://github.com/heroku/heroku-buildpack-google-chrome/pull/58